ui-showdetails-view/src/main/java/app/tivi/showdetails/details/view/ShowDetails.kt

```diff
@@ -624,27 +618,26 @@ private fun NextEpisodeToWatch(
     episode: Episode,
     onClick: () -> Unit
 ) {
-    Clickable(onClick = onClick, modifier = Modifier.ripple()) {
-        Column(
-            modifier = Modifier.paddingHV(16.dp, 8.dp)
-                .fillMaxWidth()
-                .preferredHeightIn(minHeight = 48.dp)
-                .wrapContentSize(Alignment.CenterStart)
-        ) {
-            val textCreator = ShowDetailsTextCreatorAmbient.current
+    Column(
+        modifier = Modifier.paddingHV(16.dp, 8.dp)
```

Comment:
Is this a custom modifier? Does this pre-date the horizontal/vertical overload? https://cs.android.com/androidx/platform/frameworks/support/+/androidx-master-dev:ui/ui-layout/src/main/java/androidx/ui/layout/LayoutPadding.kt;l=68?q=LayoutPadding&ss=androidx

Reply:
Thanks, removed my custom mod.